CollAction/Views/Emails/ResetPassword.cshtml

```diff
@@ -1,10 +1,17 @@
 ﻿@model string
-Hi!<br>
-<br>
-You have requested an way to reset your password on <a href="https://collaction.org">collaction.org</a>.<br>
-<br>
-Please reset your password by clicking here: <a href="@Model">link</a>.<br>
+<p>
+  Hi!
+<p>
+<p>
+  You requested a password reset on <a href="https://collaction.org">collaction.org</a>.
+</p>
+<p>
+  <a href="@Model">Click here</a> to specify a new password for your account.
+</p>
+<p>
 Warm regards,<br>
-The CollAction Team<br>
-<br>
-PS. If you've haven't requested this e-mail, you can ignore it.
\ No newline at end of file
+The CollAction Team
+</p>
+<p>
+PS. If you did not request a password reset, you can ignore this mail. However, it might be a good idea to <a href="https://howsecureismypassword.net/">check your password's strength</a>!
```

Comment:
Maybe we should add a password-strength-indicator on our registration pages instead of this line. We could add a follow-up ticket for it.